modules/nw-kuryr-migration.adoc

```diff
@@ -0,0 +1,309 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_content-type: PROCEDURE
+[id="nw-kuryr-migration_{context}"]
+= Migrating to the OVN-Kubernetes network plugin
+
+As a cluster administrator, you can change the network plug-in for your cluster to OVN-Kubernetes.
+During the migration, you must reboot every node in your cluster.
+
+[IMPORTANT]
+====
+While performing the migration, your cluster is unavailable and workloads might be interrupted.
+Perform the migration only when an interruption in service is acceptable.
+====
+
+.Prerequisites
+
+* A cluster configured with the Kuryr CNI network plug-in in the network policy isolation mode.
```

Comment:
"in the network policy isolation mode" is only specific to OpenShiftSDN, let's remove this.